### ISSUE TITLE
Run the Employee Onboarding Sample Scenario

### ISSUE THREAD

**Issue by juanpablotoledo** [SAPDocuments/Tutorials#6011]
Tutorial URL: https://developers.sap.com/tutorials/cp-starter-ibpm-employeeonboarding-5-run.html

Please specify the step you are referring to.

Hi in this part:
https://developers.sap.com/tutorials/cp-starter-ibpm-employeeonboarding-5-run.html
Step 3. I got this error when I try to upload this file EmployeeOnboardingEvents.json 
Error:
"The operation could not be performed due to missing permission"
[error.pdf](https://github.com/SAPDocuments/Tutorials/files/5380418/error.pdf)

Thank you for your help.

Juan Pablo.

**Comment by ArchanaShukla** [comment]
Hello Juan,
Can you run the booster again and try. We have added the missing role in the role collection. Reopen it if you still see the issue. 

@MichaelCzcz - Please close this issue.

**Comment by juanpablotoledo** [comment]
Thank you.

I tried again and now I am facing this error:



Juan Pablo Toledo C.
Platform & Technologies Cloud Ambassador
Carrera 9 No.115-06, Edificio Tierra Firme, piso 24
Bogota, Colombia
M +57 316 7400659

**Comment by juanpablotoledo** [comment]
Thank you.

I tried again and now I am facing this error:

[cid:image001.png@01D6A928.BF9187A0]

[cid:image002.png@01D6A928.BF9187A0]



Juan Pablo Toledo C.
Platform & Technologies Cloud Ambassador
Carrera 9 No.115-06, Edificio Tierra Firme, piso 24
Bogota, Colombia
M +57 316 7400659

**Comment by ArchanaShukla** [comment]
Hello Juan,
Can you share the images again. I cannot see anything.

**Comment by juanpablotoledo** [comment]
This the error I got:

[cid:image001.png@01D6AB9D.5A026590]

[cid:image002.png@01D6AB9D.5A026590]



Juan Pablo Toledo C.
Platform & Technologies Cloud Ambassador
Carrera 9 No.115-06, Edificio Tierra Firme, piso 24
Bogota, Colombia
M +57 316 7400659

**Comment by MichaelCzcz** [comment]
@juanpablotoledo There are still no images visible.

![image](https://user-images.githubusercontent.com/57144076/97542662-6e9e8180-19c7-11eb-9693-c1a29c31e62c.png)

**Comment by juanpablotoledo** [comment]
I send it again, thank you.

[cid:image001.png@01D6ADD6.42953690]

[cid:image002.png@01D6ADD6.42953690]





Juan Pablo Toledo C.
Platform & Technologies Cloud Ambassador
Carrera 9 No.115-06, Edificio Tierra Firme, piso 24
Bogota, Colombia
M +57 316 7400659

**Comment by MichaelCzcz** [comment]
@juanpablotoledo The images are still not visible. Are you replying via email? Please try updating this issue directly in Github.

**Comment by juanpablotoledo** [comment]
[error.pdf](https://github.com/SAPDocuments/Tutorials/files/5465276/error.pdf)

error: An error occurred while processing your request. Retry later or contact the service provider.

Archana said that it is a product error and was going to be fixed at anytime this week.

Thank you.

**Comment by juanpablotoledo** [comment]
I just replied in GitHub.

Anyway when I try to upload events from this file Employee_OnboardingEvents.json this error comes up:

error: An error occurred while processing your request. Retry later or contact the service provider.




Juan Pablo Toledo C.
Platform & Technologies Cloud Ambassador
Carrera 9 No.115-06, Edificio Tierra Firme, piso 24
Bogota, Colombia
M +57 316 7400659

**Comment by juanpablotoledo** [comment]
I just tried again and worked fine. Thank you for your help.



Juan Pablo Toledo C.
Platform & Technologies Cloud Ambassador
Carrera 9 No.115-06, Edificio Tierra Firme, piso 24
Bogota, Colombia
M +57 316 7400659



From: Toledo, Juan <juan.toledo@sap.com>
Date: Friday, 30 October 2020, 8:18 AM
To: SAPDocuments/Tutorials <reply@reply.github.com>, SAPDocuments/Tutorials <Tutorials@noreply.github.com>
Cc: Mention <mention@noreply.github.com>
Subject: Re: [SAPDocuments/Tutorials] Run the Employee Onboarding Sample Scenario (#6011)
I just replied in GitHub.

Anyway when I try to upload events from this file Employee_OnboardingEvents.json this error comes up:

error: An error occurred while processing your request. Retry later or contact the service provider.




Juan Pablo Toledo C.
Platform & Technologies Cloud Ambassador
Carrera 9 No.115-06, Edificio Tierra Firme, piso 24
Bogota, Colombia
M +57 316 7400659
